components/Card.js

```diff
@@ -5,13 +5,15 @@ import clsx from 'clsx';
 
 function Card({ backgroundImageSource, backgroundImageAltText, children }) {
   return (
-    <div className="card filter brightness-105 shadow-xl image-full">
+    <div className="card filter brightness-105 shadow-xl">
```

Comment:
```suggestion
    <div className="card filter brightness-105 shadow-xl  style={{
        boxShadow:
          'rgba(14, 30, 37, 0.1) 0px 0px 4px 0px, rgba(14, 30, 37, 0.15) 0px 0px 16px 0px;',
      }}">
```
Modifies box-shadow.

Reply:
I played with it a little and created this custom combination.

Reply:
If you'll add that then please do not use inline styles.

Reply:
Yeah, this is a great improvement to the look! We talked about moving it into a styled jsx element but I'm not sure if it has that big of a performance benefit as I had thought as per the [documentation](https://nextjs.org/docs/basic-features/built-in-css-support#css-in-js). It's an option, but this seems to work just as well. Regardless, it's a nice look.